RiotSharp/Requesters.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace RiotSharp
+{
+    internal static class Requesters
+    {
+        public static Requester StaticRequester;
+        public static Requester StatusRequester;
+        public static RateLimitedRequester GameRequester;
```

Comment:
However, I'm not sure the name `GameRequester` is quite right.
Maybe we should rename them to:

  - `StaticApiRequester`
  - `StatusApiRequester`
  - `RiotApiRequester`
  - `TournamentApiRequester`

What do you guys think?